leading-workshops/add-folks-to-2i2c-github-teams.qmd

```diff
@@ -72,18 +93,129 @@ Go back to the form and grab their email address. Send one of the following emai
 > There is a getting started guide in the NASA Earthdata Cloud Cookbook here: <https://nasa-openscapes.github.io/earthdata-cloud-cookbook/>
 >
 > We'd love to know about the kind of work you are doing on the hub, and I'll follow up in the next few months. 
-
+>
 > Best,
 
-... 
 
- 
+### 7. Register date in Form Response Sheet
+
+In the [Form Response Sheet](https://docs.google.com/spreadsheets/d/1TgokoQa-jYncYSOjeYoWYRuDAbyo4uUR-MSzUotvM00/edit?resourcekey#gid=1416760746), in Column G add the date they were added to the Hub, or if the request came via 
+email add a new row - in part so that we as admins knew if someone had already fulfilled the request. Check Column A for the date of the form request.
+
+### Directions for invitees
+
+1.  Please go to <https://github.com/nasa-openscapes>
+
+2.  Click the big yellow bar that says "you've been invited" 
+
+3.  Not seeing that bar? 
+
+    1.  Are you logged into [GitHub.com](https://github.com)?
+
+    2.  Have you shared your GitHub username via [this form](https://forms.gle/XRPsKiqaZC4bkYoi8)?
+
+    3.  You can also check your email that you use for GitHub and look for an invitation from GitHub and NASA-Openscapes
 
-## 7. Register date in Form Response Sheet
+## Adding workshop participants or Champions cohorts as a batch
 
-In the [Form Response Sheet](https://docs.google.com/spreadsheets/d/10Qg8YtUegXs9bepJzvK4UN54bX8BLXTrhxsjb2uV_6k/edit?resourcekey#gid=1824163684), in Column G add the date they were added to the Hub - in part so that we as admins knew if someone had already fulfilled the request. Rows 453+ are the most recent, but it can vary. Check Column A for the date of the form request.
+Participants in workshops run by NASA Openscapes Mentors are given Openscapes 
+2i2c JupyterHub access, as are participants in Champions Cohorts.
 
-## Directions for workshop participants
+We use a dedicated GitHub Organization - [nasa-openscapes-workshops](https://github.com/nasa-openscapes-workshops) - to manage access, with [GitHub Teams](https://github.com/orgs/nasa-openscapes-workshops/teams) for workshops and Champions Cohorts.
+
+### 1. Create a team in [nasa-openscapes-workshops](https://github.com/orgs/nasa-openscapes-workshops/teams)
+
+There are several teams in this organization; the `AdminTeam` team is for 
+members who have permission to create teams and add members to teams.
+
+- If this is for a one-off workshop, name the team `[workshop-name]-[workshop-date]`, with 
+workshop date in the format `yyyy-mm-dd`
+- If this is for a new champions cohort, name the team `nasa-champtions-yyyy`
```

Comment:
```suggestion
- If this is for a new champions cohort, name the team `nasa-champions-yyyy`
```

Reply:
Notice the Champions team name is `nasa-champions-yyyy` and their repo name would be `yyyy-nasa-champions`. Could they be named the same to reduce confusion?

Reply:
This makes sense to me!

Reply:
@jules32 what do you think?